docs/windows-requirements.md

```diff
@@ -23,8 +23,8 @@
    
    To do this, select **Desktop development for C++** under **Workloads -> Desktop & Mobile**.
 
-   <a href="/images/visualstudio.jpg" target="_blank">
-   <img src="/images/visualstudio.jpg" height="50%" width="50%">
+   <a href="/wiki/images/visualstudio.jpg" target="_blank">
+   <img src="/wiki/images/visualstudio.jpg" height="50%" width="50%">
```

Comment:
even just `images/visua..` should work without the intiial "/"

Reply:
no it doesnt

Reply:
it directs it to azerothcore.com/images then but we need azerothcore.com/wiki/images for it to work